electron/legendary/games.ts

```diff
@@ -185,22 +185,45 @@ class LegendaryGame extends Game {
   public async update() {
     this.state.status = 'updating'
     const { maxWorkers } = (await GlobalConfig.get().getSettings())
-    const workers = maxWorkers === 0 ? '' : `--max-workers ${maxWorkers}`
-    const logPath = `"${heroicGamesConfigPath}${this.appName}.log"`
-    const writeLog = isWindows ? `2>&1 > ${logPath}` : `|& tee ${logPath}`
-    const command = `${legendaryBin} update ${this.appName} ${workers} -y ${writeLog}`
-
-    try {
-      return await execAsync(command, execOptions).then((v) => {
-        this.state.status = 'done'
-        return v
+    const workers = maxWorkers === 0 ? '' : ` --max-workers ${maxWorkers}`
+    const command = `update ${this.appName}${workers} -y`.split(' ')
```

Comment:
Why you split here? Does spawn expect a array ?
Else you could make the command from the beginning an array of strings and append every thing which is defined.
Also solves the problem with the missing space for --max-workers, because you only append the argument if it is not 0.

Reply:
Yes, spawn expect and array of arguments.
Yeah but I'll still have and empty space on the command, it will be the same.